.github/ISSUE_TEMPLATE/enhancement_request.yml

```
@@ -25,28 +24,10 @@ body:
   - type: textarea
     id: enhancement-proposal
     attributes:
-      label: How would you imagine the implementation of the feature?
-      description: A clear and concise description of what you want to happen.
-    validations:
-      required: true
-  - type: checkboxes
-    attributes:
-      label: Are there any labels you wish to add?
-      description: Please search labels and identify those related to your enhancement.
-      options:
-      - label: I have added the relevant labels to the enhancement request.
-        required: true
+      label: Suggested implementation?
+      description: If you have an idea about how to implement this enhancement from a technical standpoint, feel free to share any details here.
   - type: textarea
     id: alternatives
     attributes:
       label: "Describe alternatives you've considered:"
-      description: A clear and concise description of any alternative solutions or features you've considered.
-    validations:
-      required: false
-  - type: textarea
-    id: additional-context
-    attributes:
-      label: "Additional context:"
-      description: Add any other context or screenshots about the enhancement request here.
-    validations:
-      required: false
+      description: A clear and concise description of any alternative solutions to the problem this would solve you've considered. Are there other approaches to get the same result. Any workarounds at the moment?
```

Doesn't a solution always solve it? :)
```suggestion
      description: A clear and concise description of any alternative solutions to the problem you've considered. Are there other approaches to get the same result. Any workarounds at the moment?
```